Assets/Scripts/Bahavior/Dash.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MovementBehavior
+{
+    [Header("Animations")]
+    [SerializeField] GameObject spriteObject;
+    [SerializeField] private Animator anim;
+    [Header("Stats")]
+    [SerializeField] float dashPower;
+    [SerializeField] float dashTime;
+    [SerializeField] float dashCoolDown;
+
+    //Must-have variables for movements
+    [SerializeField] IDashInput dashInput;
+    [SerializeField] TrailRenderer tr;
+
+    [SerializeField] Transform groundCheck;
+    [SerializeField] float checkRadius;
+    [SerializeField] LayerMask whatIsGround;
+
+    private bool isGrounded;
```

Comment:
@Thecoldgod04 Add this attribute into the MovementBehavior class

Reply:
Làm đê